flow-typed/debugger-html.js

```diff
@@ -206,12 +206,13 @@ declare module "debugger-html" {
    */
   declare type Source = {
     id: SourceId,
-    url?: string,
```

Comment:
is it possible that a source will not have a url? for example, a `javascript:<name>` script tag?